packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -65,7 +65,7 @@ export interface EnqueueLinksOptions extends RequestQueueOperationOptions {
      * The matching is always case-insensitive.
      * If you need case-sensitive matching, use `regexps` property directly.
```

Comment:
the last line here does not make sense in the `exclude` option context

Reply:
Ah! Making a correction now